app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -160,7 +160,9 @@ def textual_total_vms
   def textual_rps_size
     return nil if @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::EmsCluster)
 
-    textual_link(@record.resource_pools)
+    textual_link(@record.resource_pools,
+                 as: EmsCluster,
+                 link: url_for(:controller => 'ems_cluster', :action => 'show', :id => @record, :display => 'resource_pools'))
```

Comment:
Right, sorry :) fixed :) (it does feel kinda weird to call the function with different syntax than it's defined with)